resources/dicts/events/injury/general/general.json

```diff
@@ -795,5 +795,58 @@
         "cat_skill": null,
         "other_cat_trait": null,
         "other_cat_skill": null
+    },
+        {
+        "injury": "headache",
+        "camp": "any",
+        "tags": [
+            "Newleaf",
+            "Greenleaf",
+            "Leaf-fall",
+            "Leaf-bare"
+        ],
+        "event_text": "Chasing a mouse through a dense area, m_c accidentally collides with a low-hanging branch, leaving them to return to camp empty-pawed and with splitting head pain.",
+        "history_text": null,
+        "cat_trait": [
+            "ambitious",
+            "childish",
+            "playful",
+            "bold",
+            "daring",
+            "confident",
+            "adventurous",
+            "competitive",
+            "oblivious"
+        ],
+        "cat_skill": null,
+        "other_cat_trait": null,
+        "other_cat_skill": null
+    },
+    {
+        "injury": "mangled leg",
+        "camp": "any",
+        "tags": [
+            "Newleaf",
+            "Greenleaf",
+            "Leaf-fall",
+            "Leaf-bare"
+        ],
+        "event_text": "Playfully following a butterfly outside the camp, m_c accidentally steps into a rabbit burrow severely injuring {PRONOUN/m_c/poss} leg and letting out a yelp of pain.",
```

Comment:
I'd like this to be plains specific as that biome has a big rabbits focus

Reply:
👍